gui/run_manager.py

```diff
@@ -2,26 +2,32 @@
 import json
 import uuid
 import os
+import pathlib
```

Comment:
Is there a compelling reason to use this? Is there a spot where `pathlib` is preferred over `os.path`?

Reply:
`extension = pathlib.Path(filepath).suffix`

^ that was mainly just for convenience